test/test-generate-kmp.ts

```diff
@@ -26,4 +26,11 @@ test("it should generate a kmp file", async (t) => {
 
   t.assert(isKmpExist != null);
   t.assert(isModelExist != null);
+
+  //The modelID parameter determines the filename of the model.js file in the .kmp archive
+  if (isKmpExist) {
+    isKmpExist.async("string").then(function (data) {
+      t.assert(data.includes(modelID) == true);
+    });
+  }
```

Comment:
This test does _not_ exercise the funtionality implemented.

The functionality implemented affects the _filename_ of the `kmp.json` file in the zip file, however, this test asserts whether the string is contained somewhere within the `kmp.json`.

There are two steps that I'd like to see changed:

 1. revert 7a99a905c86f412acc20ffea06cd24aa6754de94
 2. create a _new_ test case (do not modify the existing one, as is done here), that calls `generateKmp()` with a _different_ modelID that what would normally be generated. Be creative here! Regardless of what the modelID is, the test should assert that the given modelID is contained in the filename of ths .js file.
 3. for extra credit, you could parse the `kmp.json` and look for the given modelID in the `files[i].name` path, where `i` is some index into the array; however, this step is optional.